src/components/banner.njk

```diff
@@ -76,5 +76,5 @@
       ],
       hidden: true
     }
-  ],  hidden: true
+  ],  hidden: false
```

Comment:
What is the effect of changing this from `true` to `false`?

Reply:
It's the default hidden state of the cookie banner.
Because the logic in the analytics package is to hide it if we don't have cookie, the default state needs to be show.